### DISCUSSION TITLE
msnoise plot distance with velocity line

### DISCUSSION THREAD

**Discussion by sujan1746** [ROBelgium/MSNoise#242] [Q&A] [answered]
I plotted the interstation distance with time lag and found some problems.

1. The graph is focused on both positive and negative part as below.
![image](https://user-images.githubusercontent.com/89066297/138394366-36c70758-c069-45fb-8142-6cbc739627ac.png)

In the manual,  the graph is focused on positive part and it shows V shape as default (below) but my graph does not.
![image](https://user-images.githubusercontent.com/89066297/138394482-3fa3f24d-9a27-4926-9e54-0ba3679f1cf2.png)

When I 'msnoise plot distance -a0.00001', it shows V shape but the distance and graph become weird.
I know '-a' means amplitude but why does the y axis focused on positive part only when I use '-a'?
![image](https://user-images.githubusercontent.com/89066297/138394597-c79b9f90-0e5d-4940-842f-338aa55cd02c.png)

2. The velocity lines should appear but they do not.
How can I see all the velocity lines along the gradient?

**Comment by ThomasLecocq** [comment, marked as answer]
I suspect you didn't input the station coordinates properly, how far are the stations from each other ?

Also, from now on, please use the https://github.com/ROBelgium/MSNoise/discussions forum for questions, Issues are meant for reporting bugs. (I transferred the question here)

**Reply by sujan1746** [reply]
Oh I see. Sorry for that. I will be more careful about that.

I input the station coordinates as below.

**Reply by ThomasLecocq** [reply]
what is the coordinate system you selected ? I bet UTM and not DEG (as your data is in DEG)

**Reply by sujan1746** [reply]
I used UTM. I see. I will try again

**Reply by ThomasLecocq** [reply]
then your stations are 10cm away from each other, good job :-)

**Reply by sujan1746** [reply]
![image](https://user-images.githubusercontent.com/89066297/138408361-4c295ab1-d22c-4afd-9e21-c11e2cb72096.png)
Finally!! Yeah It works! I will delete the coordinate fig. Thank you so much :)

**Reply by ThomasLecocq** [reply]
cool !
